inst/roxygen2-tags.yml

```diff
@@ -1,413 +1,465 @@
 - name: aliases
   description: >
-    Add additional aliases to the topic. Use `NULL` to suppress
-    the default alias automatically generated by roxygen2.
+    <p>Add additional aliases to the topic. Use <code>NULL</code> to suppress
+    the default alias automatically generated by roxygen2.</p>
   template: ' {alias}'
+  snippet: '${1:alias}'
   vignette: index-crossref
   recommend: true
 
 - name: author
   description: >
-    Topic author(s), if different from the package author(s).
+    <p>Topic author(s), if different from the package author(s).</p>
   template: ' {author}'
+  snippet: '${1:author}'
```

Comment:
I'm surprised you don't need a space here. How do you handle the tags that don't have any contents?

Reply:
https://github.com/rstudio/rstudio/blob/40f869a565da05d846366da3ebc3c94e45c9df6e/src/cpp/session/modules/SessionRCompletions.R#L132-L139 : 

```r
      snippet <- map_chr(yaml, function(.) {
         out <- .$snippet
         if (is.null(out)) {
            ""
         } else {
            paste0(" ", out)
         }
      })
```

but it's probably easier to have the space here instead. I need to check about tags sans content, maybe they can do nothing, or jump to next line. IIRC, some `template:` had `\n` before, maybe I can use that.